### PR TITLE
Docs: Update Internals > Telemetry

### PR DESCRIPTION
**Pull request by brianshumate**
- More clarification and detail around the `vault.core.leadership_lost` metric

### CODE REVIEW

File: website/pages/docs/internals/telemetry.mdx [hunk 1]
```diff
@@ -86,7 +86,7 @@ These metrics represent operational aspects of the running Vault instance.
...
 | `vault.core.leadership_setup_failed` | Duration of time taken by cluster leadership setup failures which have occurred in a highly available Vault cluster. This should be monitored and alerted on for overall cluster leadership status. | ms   | summary |
-| `vault.core.leadership_lost`         | Duration of time taken by cluster leadership losses which have occurred in a highly available Vault cluster. This should be monitored and alerted on for overall cluster leadership status.         | ms   | summary |
+| `vault.core.leadership_lost`         | Duration of time taken by cluster leadership losses which have occurred in a highly available Vault cluster. This should be monitored and alerted on for overall cluster leadership status. Any count greater than zero means that Vault experienced a leadership change and could potentially be cause for alerting.  Do note that a high mean value here is considered better than a low value, because a low value means there is leadership flapping. The measured value of this metric answers the question "how long was this server the leader, when it lost leadership?" | ms   | summary |
```
Comment (mgritter) [inline website/pages/docs/internals/telemetry.mdx:89]:
```suggestion
| `vault.core.leadership_lost`         | Measures the amount of time that the HA cluster leader was the leader, at the point when it lost leadership. This should be monitored and alerted on for overall cluster leadership status. Any count greater than zero means that Vault experienced a leadership change and could potentially be cause for alerting.  Low values are a signal of leadership flapping. | ms   | summary |
```

Comment (mgritter) [reply website/pages/docs/internals/telemetry.mdx:89]: It's really the first sentence that I feel needs editing, "Duration of time taken by cluster leadership losses" doesn't seem to mean anything.

**Comment by brianshumate** [comment]
Thanks for the great suggestion @mgritter!

**Comment by aphorise** [comment]
Request seems relevant still and the clarity in text would help.

File: website/pages/docs/internals/telemetry.mdx [hunk 2]
```diff
@@ -86,7 +86,7 @@ These metrics represent operational aspects of the running Vault instance.
...
 | `vault.core.handle_login_request`    | Duration of time taken by login requests handled by Vault core                                                                                                                                      | ms   | summary |
 | `vault.core.leadership_setup_failed` | Duration of time taken by cluster leadership setup failures which have occurred in a highly available Vault cluster. This should be monitored and alerted on for overall cluster leadership status. | ms   | summary |
-| `vault.core.leadership_lost`         | Duration of time taken by cluster leadership losses which have occurred in a highly available Vault cluster. This should be monitored and alerted on for overall cluster leadership status.         | ms   | summary |
+| `vault.core.leadership_lost`         | The total duration that a HA cluster node maintained leadership as reported at the last time of loss. Any count greater than zero means that a leadership change has occurred. Continuing changes or reports of low value could be cause for monitoring alerts as they'd typically imply ongoing flapping of leadership that may be rotating between nodes. | ms   | summary |
```
Comment (taoism4504) [inline website/pages/docs/internals/telemetry.mdx:89]:
```suggestion
| `vault.core.leadership_lost`         | The total duration that a HA cluster node maintained leadership as reported at the last time of loss. Any count greater than zero means that a leadership change has occurred. Continuing changes or reports of low value could be a cause for monitoring alerts as they would typically imply ongoing flapping of leadership that may be rotate between nodes. | ms   | summary |
```

**Review by taoism4504 — APPROVED** [review]

**Pull request merged**